docs/platforms/javascript/common/configuration/integrations/http.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Http
+description: "Capture spans & breadcrumbs for http requests. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.gcp-functions
+  - javascript.koa
+  - javascript.electron
+---
+
+_Import name: `Sentry.httpIntegration`_
+
+This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `httpIntegration` does two things:
+1. It captures spans for HTTP requests.
+2. It captures breadcrumbs for HTTP requests.
```

Comment:
```suggestion
1. It captures breadcrumbs for HTTP requests.
2. When the [`tracing`](#tracing) option is enabled, it captures spans for outgoing HTTP requests.
```
I would reorder these to align with the ordering of options below. 

It also sounds like breadcrumbs is enabled by default but span tracing isn't, so I would clarify that here as well. Since line 17 said this integration is enabled by default, the manual opt-in for tracing isn't obvious.